src/components/code.tsx

```diff
@@ -4,19 +4,29 @@ import { Lang, Theme } from 'shiki'
 const w3upExample = `
 import * as Client from '@web3-storage/w3up-client'
 import { filesFromPaths } from 'files-from-path'
+import * as UcantoClient from '@ucanto/client'
+import { HTTP } from '@ucanto/transport'
+import * as CAR from '@ucanto/transport/car'
 
 const client = await Client.create()
 
 // first time setup!
 if (!Object.keys(client.accounts()).length) {
   // waits for you to click the link in your email to verify your identity
   const account = await client.login('you@example.org')
-  // create a space for your uploads
-  const space = await client.createSpace('lets-go')
+  // connect to the Gateway that will serve your content
+  const storachaGateway = UcantoClient.connect({
+      id: id,
+      codec: CAR.outbound,
+      channel: HTTP.open({ url: new URL('https://freeway.dag.haus') }),
+  })
+  // create a space for your uploads and authorize the gateway
+  const space = await client.createSpace('lets-go', {
+    account, // associate this space with your account & configure recovery
```

Comment:
do you think it would be possible to pick smart defaults for `account` and `authorizeGatewayServices` inside `createSpace` or is that infeasible here? It feels like most people will want to publish their data using our gateway, so it would be nice for that to be the default and to show people how they can customize that in these docs

Reply:
@travis, It would be great to introduce a smart default for `authorizeGatewayServices`, but this can be addressed in a future iteration if you're okay with that. As for the `account`, I believe we shouldn't set a default since it serves as the recovery account. It's safer to require the user to explicitly define it.

Reply:
@travis - just realized that it was a simple change to add the default for the `authorizeGatewayServices`: https://github.com/storacha/w3up/pull/1604